managed/Makefile

```diff
@@ -15,6 +15,7 @@ PMM_RELEASE_VERSION ?= $(shell git describe --always --dirty | cut -b2-)
 PMM_RELEASE_TIMESTAMP ?= $(shell date '+%s')
 PMM_RELEASE_FULLCOMMIT ?= $(shell git rev-parse HEAD)
 PMM_RELEASE_BRANCH ?= $(shell git describe --always --contains --all)
+export PATH := $(PMM_RELEASE_PATH):$(PATH)
```

Comment:
Ok, here is the problem. Actually we had such export everywhere some time ago, and then we dropped it. Trick is that your targets may work even if you forgot to invoke `make init`. Some outdated tool will be taken from your PATH.

Reply:
That's possible, I'll check if it's possible to use absolute path in generate comments.